docs/reference/garbage-collection.md

```diff
@@ -283,11 +283,19 @@ This utility is a Spark application that uses [distCp](https://hadoop.apache.org
 * Backup: copy expired objects from your repository's storage namespace to an external location before running GC in [sweep only mode](#sweep-only-mode).  
 * Restore: copy objects that were hard-deleted by GC from an external location you used for saving your backup into your repository's storage namespace.
 
+### Job options
+
+- This will only work with Hadoop versions > 3.1.3, meaning only for our hadoop 3 client.
```

Comment:
```suggestion
- This will only work with Hadoop versions > 3.1.3, and so require a Hadoop 3 client.
```
(Both capitalization and to note that "Hadoop 3" itself is not enough!)

Reply:
thank you! done